scrapers/covid-19/govScrapers/getVietnam.js

```diff
@@ -2,33 +2,44 @@ const axios = require('axios');
 const cheerio = require('cheerio');
 const logger = require('../../../utils/logger');
 
-const columns = ['city', 'cases', 'beingTreated', 'recovered', 'deaths'];
-
 const mapRows = (_, row) => {
-	const city = { updated: Date.now() };
+	const columns = ['_', '_', '_', 'location', 'status'];
```

Comment:
There is actually 6 columns, so it should be: 
`['_', '_', '_', 'location', 'status', _]`